### ISSUE TITLE
[A11y] NVDA does not read the position number of "Online Cards"

### ISSUE THREAD

**Issue by HuihuiWu-Microsoft** [MadeWithCards/AdaptiveCardsStudio#7]
Repro steps:
1. Tab till ""Preview and Debug Adaptive Cards"" and hit enter
2. Observed that screen reader not announcing position number of "Online Cards"

Expected:
Screen reader should announce position number of "Online Cards" as "Two of Two"

**Comment by DeeJayTC** [comment]
Thanks for reporting this, ill see what i can do!

**Comment by HuihuiWu-Microsoft** [comment]
Hi @DeeJayTC, the issue has been open for a while. I'd like to confirm about the schedule when this will be fixed. Thanks!

**Comment by DeeJayTC** [comment]
Hi @HuihuiWu-Microsoft It should hopefulle be fixed in 1.2.0 which i just published

**Comment by HuihuiWu-Microsoft** [comment]
> efulle be fixed in 1.2.0 which i just pub

Thanks @DeeJayTC, I've verified and everything works well!
